pkg/cmd/auth/status/status.go

```diff
@@ -97,48 +97,61 @@ func statusRun(opts *StatusOptions) error {
 			statusInfo[hostname] = append(statusInfo[hostname], fmt.Sprintf(x, ys...))
 		}
 
-		if err := shared.HasMinimumScopes(httpClient, hostname, token); err != nil {
-			var missingScopes *shared.MissingScopesError
-			if errors.As(err, &missingScopes) {
-				addMsg("%s %s: the token in %s is %s", cs.Red("X"), hostname, tokenSource, err)
-				if tokenIsWriteable {
-					addMsg("- To request missing scopes, run: %s %s\n",
-						cs.Bold("gh auth refresh -h"),
-						cs.Bold(hostname))
+		scopesHeader, err := shared.GetScopes(httpClient, hostname, token)
+		if err != nil {
+			failed = true
+			addMsg("%s %s: api call failed: %s", cs.Red("X"), hostname, err)
+		} else {
```

Comment:
adding this else pushed everything to the right, but I think is necessary to keep with the current pattern of setting `failed = true` and continuing execution. This function would benefit from a refactor, unfortunately I'm not able to take that on at this time